modules/odc-grouping-multiple-components.adoc

```diff
@@ -11,20 +11,25 @@ The following procedure adds a MariaDB database service to an existing applicati
 
 .Prerequisites
 
-* Ensure that you have created and deployed a Node.js application on {product-title} using the *Developer* perspective.
+* Ensure that you have created and deployed minimum two or more components on {product-title} using the *Developer* perspective.
 
 .Procedure
 
-. Create and deploy the MariaDB service to your project:
-.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all available services in the *Developer Catalog*.
-.. Select *Databases* to list all the database services and then click *MariaDB* option to see the details for the service.
-.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.
+. To add a service to the existing application group, press kbd:[Shift]+ drag it to the existing application group
 
-. To add the MariaDB service to the existing application group, press kbd:[Shift]+ drag the *mariadb* pod to the application group that contains the Node.Js component. The MariaDB service is added to the existing application group.
-
-. Click the MariaDB service node and confirm that the `app.kubernetes.io/part-of=<application-name>` label is in the *Labels* section in the *Overview* Panel. Adding a component to an application group automatically adds the required labels to the component.
+. Dragging a component and adding it to an application group automatically adds the required labels to the component.
 +
 .Application grouping
 image::odc_app_grouping_label.png[]
 
+Alternatively, you can also add the component to an application as follows:
+
+. Click on the service pod to see the *Overview* panel to the right.
+
+. Click the *Actions* drop-down menu on the upper right of the panel and select *Edit Application Grouping*.
+
+. In the *Edit Application Grouping* dialog box, click the *Application* drop-down list, and select the appropriate application group.
```

Comment:
```suggestion
. In the *Edit Application Grouping* dialog box, click the *Application* drop-down list, and select an application group.
```
Minimalism. Does "appropriate" add meaningful information to this step, or does it potentially raise doubts in the user's mind? For example, is it always obvious to the user which application group they should select, or might they ask whether there are appropriate and inappropriate items?

Reply:
Need some clarity on this one too.

Reply:
Here's how I like to think about words like this: Does the user know what "appropriate" means? Does the content explain what is appropriate and what isn't, or is this just a filler word? If I can remove the word and the phrase seems clearer without it, then it's a filler word that shouldn't be there.

Reply:
Got it thank